crates/red_knot_wasm/Cargo.toml

```diff
@@ -19,10 +19,10 @@ doctest = false
 default = ["console_error_panic_hook"]
 
 [dependencies]
-red_knot_python_semantic = { workspace = true }
 red_knot_project = { workspace = true, default-features = false, features = ["deflate"] }
 
 ruff_db = { workspace = true, default-features = false, features = [] }
+ruff_python_parser = { workspace = true, features = ["wasm-bindgen"] }
```

Comment:
I think I'd also prefer to keep the wasm wrapper to avoid having the parser depend on wasm-bindgen just for that.